### PR TITLE
Set up react web frontend

### PR DESCRIPTION
**Pull request by Thykof**
with:
- [x] vite,
- [x] react-ts,
- [x] tailwind,
- [x] storybook,
- [x] our eslint and prettier config
- [x] husky
- [x] lintstaged
- [x] react router
- [x] integrate this new frontend to golang server
  - [x] serve index.html
  - [x] serve assets (fonts, svg...) 

latter:
- [x] unit test
- [x] pre push npm run test

### CODE REVIEW

**Review by 0xMazout — APPROVED** [review]

**Comment by 0xMazout** [comment]
Just an advise , maybe we should wait the meetings today at 11am , where @mazmassa will showcase what will be the new standards.

You will have to make some changes on the Tests part

File: web-frontend/src/components/AccountListItem/AccountListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { Amount } from '@massalabs/react-ui-kit';
+
+export interface AccountListItemProps {
+  name: string;
+  amount: number;
+}
+
+export default function AccountListItem(props: AccountListItemProps) {
+  return (
+    <div className="flex flex-row justify-between align-center py-2 px-3 gap-48">
```
Comment (mazmassa) [inline web-frontend/src/components/AccountListItem/AccountListItem.tsx:12]: This is too much! We must rethink that: `flex flex-row justify-between align-center py-2 px-3 gap-48`

It's just not maintainable, hard to follow and understand. So we are adding something to make our life hard.

Comment (Thykof) [reply web-frontend/src/components/AccountListItem/AccountListItem.tsx:12]: what do you suggest to simplify this? 

I found this way: https://stackoverflow.com/a/73899407/5903242 but we must keep in mind that we want to share those aliases from the ui-kit

**Review by mazmassa — APPROVED** [review]
I guess its the code we did in the call. So, LGTM!

File: api/server/restapi/configure_massa_wallet.go [hunk 2]
```diff
@@ -82,11 +82,18 @@ func configureAPI(api *operations.MassaWalletAPI) http.Handler {
 			return middleware.NotImplemented("operation operations.RestAccountSignOperation has not yet been implemented")
 		})
 	}
+	// route for the legacy wallet frontend
 	if api.WebHandler == nil {
 		api.WebHandler = operations.WebHandlerFunc(func(params operations.WebParams) middleware.Responder {
 			return middleware.NotImplemented("operation operations.Web has not yet been implemented")
 		})
 	}
+	// route for the ReactJS wallet frontend
```
Comment (peterjah) [inline api/server/restapi/configure_massa_wallet.go:91]: i don't really understand why you add this... its actually implemented

**Review by peterjah — APPROVED** [review]

File: api/server/restapi/configure_massa_wallet.go (hunk 2 above)
Comment (Thykof) [reply api/server/restapi/configure_massa_wallet.go:91]: I will delete it in another PR, thanks

**Comment by SlnPons** [comment]
Hey @Thykof can you help to test this ticket please? 
Can we consider that if the integration of the 1st wallet screen is a success it's working?
